src/styles/blocks/product_card/_card.scss

```diff
@@ -0,0 +1,141 @@
+$card-width: 200px;
+$card-height: 408px;
+$other-width: 166px;
+$button-height: 40px;
+$bottom-margin-image: 40px;
+$bottom-margin-name: 4px;
+$bottom-margin-box: 16px;
+$bottom-margin-reviews: 24px;
+$bottom-margin-price: 16px;
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
general styles for all elements/ resetting default styles or overwritting them should be in reset.scss or normalize.scss